genrules.cpp

```diff
@@ -171,21 +171,21 @@ void Game::WriteFactionTypeDescription(std::ostringstream& buffer, Faction &fac)
 				buffer << "perform tax or trade in " << nma << " " << plural(nma, "region", "regions");
 			}
 			
-			if (Globals->FACTION_ACTIVITY == FactionActivityRules::MARTIAL_MERGED) { 
+			if (Globals->FACTION_ACTIVITY == FactionActivityRules::MARTIAL_MERGED) {
 				buffer << "perform tax and trade in " << nma << " " << plural(nma, "region", "regions");
 			}
 		}
 
 		count++;
 	}
-
+/*
```

Comment:
why did you commented this part?

Reply:
I commented that out a couple of days ago...When it generated the rules it was printing "as well as 5 apprentices" twice. Once with the information about mages and a second time at the end of the description block.  Commenting out this code got rid of the second time.  I left it in but commented out as I was not sure what effect that change would have under different rules configurations.  It works properly with current configuration, but haven't had time to make sure that the output is correct with Martial (not Martial Merged) and original War/Trade/Magic rules.